script.js

```diff
@@ -0,0 +1,86 @@
+
+const newNote = document.querySelector("#recipeCard") //text area
+const newRecipe = document.querySelector(".newRecipe") //button
+const noteForm = document.querySelector("#notes-area") //form
+const deleteRecipe = document.querySelector(".deleteRecipe") //button
+
+
+//const newRecipe = document.getElementById("#newRecipe") 
+
+window.addEventListener('load', function () {
+    fetch('http://localhost:3000/notes/')
+        .then(response => response.json())
+        //when the page loads, get a new 'recipe card to type on'
+        .then(recipeCard => {
+            console.log(recipeCard)
+            const recipeCardEls = createNote.map(newNote => createRecipeCardEl(newNote))
```

Comment:
The way you wrote this makes me think you aren't sure what is happening in this second `.then` -- let's chat about this when we meet! What is `recipeCard` when you see it in the console? Why do we need that? Understanding that part will help you fix line 16. `createNote` is not defined here (I see a function with that name defined below though, so maybe you had that in mind). So let's think through what we need to do here and then how we need to do it.